_commands/tools/less.md

```diff
@@ -0,0 +1,54 @@
+---
+---
+
+less
+-------
+
+`less` is used to display contents of a file, as well as allow navigation. Its name is derived from being the _opposite of_ `more`.
+
+~~~ bash
+$ less file.txt
+
+These are the contents of "file.txt".
+
+The output will actually take up the entire screen, and you can navigate using Vim-like commands.
```

Comment:
Maybe either link to vim or give a brief description of the commands? Maybe briefly dive into `q`, scrolling and `/` for searching?